java/ql/src/Telemetry/ExternalAPI.qll

```diff
@@ -0,0 +1,40 @@
+import java
+import APIUsage
+private import experimental.semmle.code.java.Logging
```

Comment:
Avoid dependencies from standard libraries onto experimental (other way around is OK).

Reply:
Done. Including all logging calls for now s I think it doesn't hurt to include them given we know have better metadata to distinguish between relevant results.